helm_remote/Tiltfile

```diff
@@ -98,6 +98,8 @@ def helm_remote(chart, repo_url='', repo_name='', release_name='', values=[], se
     else:
         k8s_yaml(yaml)
 
+    return yaml
```

Comment:
I'm guessing the preference is to avoid modifying existing files, was trying to avoid needing to add a large blob of yaml to this repo at the same time.

This shouldn't break anyone?

Reply:
Cool